src/python/pants/jvm/dependency_inference/artifact_mapper_test.py

```diff
@@ -53,10 +51,15 @@ def rule_runner() -> RuleRunner:
             QueryRule(Addresses, [DependenciesRequest]),
             QueryRule(ThirdPartyPackageToArtifactMapping, []),
         ],
-        target_types=[JavaSourcesGeneratorTarget, JunitTestsGeneratorTarget, JvmArtifactTarget],
+        target_types=[
+            JavaSourceTarget,
+            JavaSourcesGeneratorTarget,
+            JunitTestsGeneratorTarget,
+            JvmArtifactTarget,
+        ],
     )
     rule_runner.set_options(
-        args=[NAMED_RESOLVE_OPTIONS, DEFAULT_RESOLVE_OPTION], env_inherit=PYTHON_BOOTSTRAP_ENV
+        args=[NAMED_RESOLVE_OPTIONS, DEFAULT_RESOLVE_OPTION], env_inherit={"PATH"}
```

Comment:
Was this related to this change? The `PYTHON_BOOTSTRAP_ENV` is necessary on hosts that use `pyenv` (in order to locate the Python 3.6 which is used to `gunzip` the coursier archive), so this isn't equivalent.

Reply:
Oh, hm. Will revert. I assumed it was bad copy pasta from python tests